src/Components/Facility/BedManagement.tsx

```diff
@@ -76,68 +76,79 @@ const BedRow = (props: BedRowProps) => {
         handleCancel={handleDeleteCancel}
         handleOk={handleDeleteConfirm}
       />
-      <div
-        key={id}
-        className="w-full items-center justify-between border-b py-4 lg:flex"
-      >
-        <div className="mt-2 space-y-2 px-4 lg:w-3/4">
-          <div className="flex flex-col sm:flex-row">
-            <p
-              className="inline break-words text-xl capitalize"
-              id="view-bed-name"
-            >
-              {name}
-            </p>{" "}
-            &nbsp;
-            <div id="view-bedbadges">
-              {LOCATION_BED_TYPES.find((item) => item.id === bedType) && (
-                <p className="mb-1 inline-flex w-fit items-center rounded-md bg-blue-100 px-2.5 py-0.5 text-sm font-medium capitalize leading-5 text-blue-800">
-                  {LOCATION_BED_TYPES.find(
-                    (item) => item.id === bedType
-                  )?.name?.slice(0, 25) + (bedType.length > 25 ? "..." : "")}
-                </p>
-              )}
-              <p
-                className={`${
-                  isOccupied
-                    ? "bg-warning-100 text-warning-600"
-                    : "bg-primary-100 text-primary-600"
-                } mb-1 ml-1 inline-flex w-fit items-center rounded-md px-2.5 py-0.5 text-sm font-medium capitalize leading-5`}
-              >
-                {isOccupied ? "Occupied" : "Vacant"}
-              </p>
+      <div className="parent  m-2 flex  lg:flex-row">
+        <div
+          key={id}
+          className="  child m-2 items-center justify-between rounded-xl  border border-gray-300  bg-white py-4 hover:border-green-700 "
```

Comment:
Use `primary` shade instead of `green`.

```suggestion
          className="  child m-2 items-center justify-between rounded-xl  border border-gray-300  bg-white py-4 hover:border-primary-700 "
```